features/admin.connections.v1/components/create/authenticator-create-wizard-factory.tsx

```diff
@@ -330,6 +337,27 @@ export const AuthenticatorCreateWizardFactory: FC<AuthenticatorCreateWizardFacto
                     />
                 );
 
+            case CommonAuthenticatorConstants.CONNECTION_TEMPLATE_IDS.CUSTOM_AUTHENTICATION:
+                if (isFeatureEnabled(customAuthFeatureConfig, "identityProviders.customAuthentication")) {
```

Comment:
Let's define the `identityProviders.customAuthentication` as a feature identifier in /constants and refer it here.

This is done to organize the feature identifiers for each feature in a central location.

Eg:

```suggestion
                if (isFeatureEnabled(
                    customAuthFeatureConfig,  ConnectionUIConstants.FEATURE_DICTIONARY.CUSTOM_AUTHENTICATION)) {
```

Ref. definition of feature dictionary object: https://github.com/wso2/identity-apps/blob/master/features/admin.groups.v1/constants/group-constants.ts#L40